### DISCUSSION TITLE
绑定账号是米游社 id 而不是 uid 感觉确实不是一个很好的设计

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#962] [General]
这个功能现在印象里只用来显示 `米游社` 的用户名称，其他时候没有用。

大部分用户也不会特意去记自己的米游社 id，倒是对 uid 记得比较熟。

我觉得可以更改 bot 绑定提示为【绑定uid】，将绑定的行为由【只能够绑定米游社账号】改成【9 位数账号优先识别为游戏内 uid】，不满 9 位的才识别成米游社 id

这样会更方便一些

**Comment by BTMuli** [comment]
> 我觉得可以更改 bot 绑定提示为【绑定uid】，将绑定的行为由【只能够绑定米游社账号】改成【9 位数账号优先识别为游戏内 uid】，不满 9 位的才识别成米游社 id

我的米游社id 249066520就是9位的('◇'｀)

**Reply by mark9804** [reply]
啊这，米忽悠真自由啊

**Comment by Arondight** [comment]
我忘了为什么要这么做了，但是我记得必须要这么做
